terraform/modules/pre-auth-lambda/outputs.tf

```diff
@@ -1,3 +1,11 @@
 output pre_auth_lambda {
   value = aws_lambda_function.lambda_pre_auth
 }
+
+output dynamodb_table_user_arn {
+  value = aws_dynamodb_table.dynamodb_table_user.arn
+}
+
+output dynamodb_table_user_name {
+  value = aws_dynamodb_table.dynamodb_table_user.name
+}
```

Comment:
```suggestion
output dynamodb_table_user {
  value = {
      arn = aws_dynamodb_table.dynamodb_table_user.arn
      name = aws_dynamodb_table.dynamodb_table_user.name
    }
}
```

Reply:
formatting is a little off there due to editing in comment box, but `terraform fmt` will fix

Reply:
This is done thanks.